Complete the result of coverage in the report

apeinot opened the pull request:
Add coverage result
Add new tests link
This should resolve #96

apeinot committed:
Complete the result of coverage in the report

Add coverage result
Add new tests link
This should resolve #96

apeinot committed:
Correct a few typos and add info about decodeFast()

This is part of #96

Jonagoo commented on report.md:
```diff
@@ -257,13 +257,55 @@ its output?
-Test cases added:
-
-git diff ...
+The obtained results from our tool are close to what we can observe in Cobertura report (green lines and red lines of coverage in functions as there are coverage pourcentage for functions) so it's pretty accurate. However, if the code is modified (or refactored) then some flags will need to be changed before running the tool again.
```
percentage instead of pourcentage

Jonagoo commented on report.md:
```diff
@@ -257,13 +257,55 @@ its output?
+
+#### Report of new coverage:  
+(run `mvn test|grep coverage`on *coverage2* branch to get it)  
+(Table only for the functions where new test cases are added / new tests have been added to AllTestCase test suite so there executed in any case)  
```
they're executed instead of there executed

apeinot committed:
Add coverage and files for the new tests

Complete the coverage results according to the new test
This should complete #96

apeinot commented:
The typos should be fixed now
